packages/@angular/cli/tasks/update.ts

```diff
@@ -0,0 +1,30 @@
+const Task = require('../ember-cli/lib/models/task');
+import SchematicRunTask from './schematic-run';
+
+export interface UpdateTaskOptions {
+  dryRun: boolean;
+  force: boolean;
+}
+
+export const UpdateTask: any = Task.extend({
+  run: function(schematic: string, options: UpdateTaskOptions): Promise<any> {
+    const [collectionName, schematicName] = schematic.split(':');
+
+    const schematicRunTask = new SchematicRunTask({
```

Comment:
Could potentially use `SchematicRunTask` directly.  Unless there's more planned.

Reply:
Possibly, but that layer is consistent with the other command/task combinations. I'm indifferent about this.